lib/get_pid.c

```diff
@@ -51,6 +53,21 @@ int get_pidfd_from_fd(const char *pidfdstr)
 		return -1;
 	}
 
+	if (stat("/proc/self/uid_map", &st) < 0) {
+		return -1;
+	}
+
+	proc_st_dev = st.st_dev;
+	proc_st_rdev = st.st_rdev;
+
+	if (fstat(proc_dir_fd, &st) < 0) {
```

Comment:
there's a copy'n'paste error here. It should be `fstat(val, &st)`

Reply:
Yup, updated, thanks.